setup.py

```diff
@@ -2,7 +2,7 @@
 
 import os
 import re
-#from distutils.core import setup
+# from distutils.core import setup
```

Comment:
Looked into it, it appears the next line "from setuptools import setup" is what replaced this line. It appears this was a change when porting to Python 3. 
Removing this commented out code.